inject.js

```diff
@@ -361,8 +361,7 @@ function defineVideoController() {
         p.insertBefore(fragment, p.firstChild);
         break;
       case location.hostname == "tv.apple.com":
-        // insert after parent for correct stacking context
-        this.parent.getRootNode().querySelector(".scrim").prepend(fragment);
+        // pass-through, the latest version of AppleTV works with default behavior
```

Comment:
Can we remove this branch? No purpose, right?

Reply:
sounds good

Reply:
I ended up modifying it so that the hover works too (before it would show and you could use hotkeys but not modify the UI directly)